### ISSUE TITLE
illegal comment in json file flagged by pre-commit check-json

### ISSUE THREAD

**Issue by kbroch-rivosinc** [riscv-software-src/riscv-unified-db#267]
This might be a don't care but currently this file: 
https://github.com/riscv-software-src/riscv-unified-db/blob/main/.devcontainer/devcontainer.json#L12
is marked as `.json` and therefore can't have comments.

```bash
❯ pre-commit run --all-files check-json
check json...............................................................Failed
- hook id: check-json
- exit code: 1

.devcontainer/devcontainer.json: Failed to json decode (Expecting property name enclosed in double quotes: line 12 column 3 (char 359))
```

If this format is acceptable for devcontainers I'd suggest renaming it `.json5` 
If not we can just ignore it when check-json is put in place.

**Comment by dhower-qc** [comment]
I'm not sure we can rename it to `.json5`. According to the [devcontainer spec](https://containers.dev/implementors/spec/#devcontainerjson), this file must be named `devcontainer.json`. Further, the spec says it is "JSON with comments," not specifically JSON5.

**Comment by kbroch-rivosinc** [comment]
Ok
In the short term, this file can just be removed from `check-json` hook. 

In the long term, I'll see about getting a `check-json-with-comments` in pre-commit-hooks repo where it has already been discussed here:
* https://github.com/pre-commit/pre-commit/issues/1320
* https://github.com/pre-commit/pre-commit-hooks/issues/395

**Comment by AFOliveira** [comment]
@kbroch-rivosinc is this fixed?

**Comment by kbroch-rivosinc** [comment]
yes, we excluded the devcontainer jsonc file: https://github.com/riscv-software-src/riscv-unified-db/blob/main/.pre-commit-config.yaml#L22
